src/dhcp_device.cpp

```diff
@@ -398,7 +398,7 @@ static void read_rx_callback(int fd, short event, void *arg)
             continue;
         }
         std::string intf(interfaceName);
-        context = interface_to_dev_context(devices, intf);
+        context = find_device_context(devices, intf);
```

Comment:
I think we should update here https://github.com/sonic-net/sonic-dhcpmon/blob/master/src/dhcp_device.cpp#L324

Reply:
Yes, updated